packages/components/src/internal/actions.ts

```diff
@@ -245,19 +253,7 @@ export function exportRows(type: EXPORT_TYPES, exportParams: Record<string, any>
     }
 
     const url = buildURL(controller, action, undefined, { returnUrl: false });
-
-    // POST a form
-    const form = $(`<form method="POST" action="${url}">`);
-    $.each(params, function (k, v) {
-        const safeValue = quoteEncodedValue(v);
-        if (safeValue instanceof Array) {
-            safeValue.forEach(val => {
-                form.append($(`<input type="hidden" name="${k.toString()}" value="${val}">`));
-            });
-        } else form.append($(`<input type="hidden" name="${k.toString()}" value="${safeValue}">`));
-    });
-    $('body').append(form);
-    form.trigger('submit');
+    Ajax.request({ url: url.toString(), method: 'POST', form, downloadFile: true });
```

Comment:
nit: `url` is already a string so no need to call `toString()`.

Reply:
Oh good catch, my eyes saw `AppURL.buildUrl` out of habit.